engine/source/elements/beam/beam_fail.F

```diff
@@ -0,0 +1,201 @@
+Copyright>        OpenRadioss
+Copyright>        Copyright (C) 1986-2023 Altair Engineering Inc.
+Copyright>
+Copyright>        This program is free software: you can redistribute it and/or modify
+Copyright>        it under the terms of the GNU Affero General Public License as published by
+Copyright>        the Free Software Foundation, either version 3 of the License, or
+Copyright>        (at your option) any later version.
+Copyright>
+Copyright>        This program is distributed in the hope that it will be useful,
+Copyright>        but WITHOUT ANY WARRANTY; without even the implied warranty of
+Copyright>        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+Copyright>        GNU Affero General Public License for more details.
+Copyright>
+Copyright>        You should have received a copy of the GNU Affero General Public License
+Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Copyright>
+Copyright>
+Copyright>        Commercial Alternative: Altair Radioss Software
+Copyright>
+Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+Chd|====================================================================
+Chd|  BEAM_FAIL                        source/elements/beam/mulawp.F 
+Chd|-- called by -----------
+Chd|        PMAIN3                        source/elements/beam/pmain3.F 
+Chd|-- calls ---------------
+Chd|====================================================================
+      SUBROUTINE BEAM_FAIL(ELBUF_STR     ,NUMMAT  ,NUMGEO  ,NPROPM  ,
+     .                  NPROPMI ,NPROPG  ,SBUFMAT ,SNPC    ,STF     ,
+     .                  NEL     ,NPT     ,IMAT    ,JTHE    ,TEMPEL  ,
+     .                  PID     ,NGL     ,PM      ,IPM     ,GEO     ,
+     .                  OFF     ,EPSP    ,BUFMAT  ,NPF     ,TF      ,
+     .                  EINT    ,TIME    )
+C-----------------------------------------------
+C   M o d u l e s
+C-----------------------------------------------
+      USE ELBUFDEF_MOD
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+C-----------------------------------------------
+C   C o m m o n   B l o c k s
+C-----------------------------------------------
+#include  "units_c.inc"
```

Comment:
Hi @MaciekWronski 

It is not allowed to use commons, can you pass by arguments the file units you need ( removing units_c.inc ) ?

Thanks,
Sébastien

Reply:
done